### PR TITLE
chore: update node-sdk

### PR DESCRIPTION
**Pull request by joebowbeer**
## What was changed

Update position-node to node-sdk 0.4.2

## Checklist

[x] Followed steps in README and was able to switch between Java and Node workflows

### CODE REVIEW

File: position-node/src/interfaces.ts [hunk 1]
```diff
@@ -1,6 +1,4 @@
-import { Workflow } from '@temporalio/workflow';
-
-export interface GameInfo {
+export type GameInfo = {
```
Comment (bergundy) [inline position-node/src/interfaces.ts:1]: Why did you change this to type?

**Review by bergundy — APPROVED** [review]
Thanks!

File: position-node/src/interfaces.ts (hunk 1 above)
Comment (joebowbeer) [reply position-node/src/interfaces.ts:1]: Naively, only for consistency with the workflow type in your new examples - which exist in a file named interfaces but, I observed, were recently changed from an interface to a type.

According to the following blog, interface is _preferable_ for GameInfo? Should I change it back?

https://blog.scottlogic.com/2021/06/24/types-vs-interfaces.html

If there's any practical difference, especially in the temporal context, it might be helpful to add a note in the docs.

**Review by tsurdilo — APPROVED** [review]

File: position-node/src/interfaces.ts (hunk 1 above)
Comment (bergundy) [reply position-node/src/interfaces.ts:1]: The differences are listed in the post you linked, nothing special in the temporal context.
I would change it back but it's really not a show stopper.

Comment (joebowbeer) [reply position-node/src/interfaces.ts:1]: @bergundy Fixed in #6
